syntax_checkers/scala/ensime.vim

```diff
@@ -0,0 +1,35 @@
+let s:save_cpo = &cpo
```

Comment:
Could you please place this file in another folder? It would be better to have a folder called `plugin-integrations` or `external-integrations` and another folder inside named `syntactic` to make explicit that this code serves only one purpose, that is, be an interface for this plugin :smiley: